lib/paper_trail/reifier.rb

```diff
@@ -55,20 +56,10 @@ def reify(version, options)
         reify_attributes(model, version, attrs)
         model.send "#{model.class.version_association_name}=", version
         reify_associations(model, options, version)
-        model
```

Comment:
I think I'd rather keep the explicit return value here, instead of moving it to `reify_associations`.